docs/other_schemas.rst

```diff
@@ -125,3 +125,72 @@ this command::
     prophyc --isar --python_out . test.xml
 
 generates identical codec to one from previous example.
+
+Mixing Isar and Sack
+--------------------
+
+Since version ``1.1.0`` there is a possibility to implicitly include Isar definitions to
+Sack schema compilation. That's a useful if the Sack code (c++) depends on many definitions originated in
+Isar XMLs. Probably in normal way these are supplied in C++ form by monstrual build system.
+This feature allows to skip the 'build system' stage and supplement definitions directly from Isar XMLs.
+
+Having Isar code defining ``Test`` type from previous subchapter (the ``test.xml`` file) you can successfully
+compile it with such a Sack code::
+
+   // mixing.hpp
+   struct MixingTest
+   {
+       int field_a;
+       Test field_b;
+   };
+
+.. note::
+
+  There is missing definition of ``Test`` and that would fail to compile with any c/c++ compiler. No includes
+  are defined, nor exising in filesystem, besides the ``test.xml``. It's up to user to prepare c/c++ file, e.g.
+  remove all problematic inclusions. That's why we call it `implicit`.
+
+Such a prophyc call::
+
+  prophyc --sack --include_isar test.xml --python_out . mixing.hpp
+
+Will generate two python files.
+
+Comming from Isar: ``test.xml`` -> ``test.py``::
+
+    # test.py
+    import prophy
+
+    class Test(prophy.with_metaclass(prophy.struct_generator, prophy.struct)):
+        _descriptor = [('x_len', prophy.u32),
+                       ('x', prophy.array(prophy.u32, bound = 'x_len'))]
+
+And from the sack code ``mixing.hpp`` -> ``mixing.py``::
+
+    # mixing.py
+    import prophy
+
+    from test import *
+
+    class MixingTest(prophy.with_metaclass(prophy.struct_generator, prophy.struct)):
+        _descriptor = [('field_a', prophy.i32),
+                       ('field_b', Test)]
+
+.. note::
+
+  How it wotks from the kitchen side:
+
+  * ``text.xml`` has been compiled to prophy model object.
+  * A supplementary file ``isar_supplementary_defs.h`` has been created in temporary directory.
```

Comment:
This doesn't work:
```python
(Pdb++) cursor.spelling
't'
(Pdb++) cursor.type.spelling
'int'
```
But this works:
```python
(Pdb++) cursor.extent.start.offset, cursor.extent.end.offset
(26, 32)
>>> open("x.hpp").read()[26:32]
'Test t'
>>> spelling = 'x'
>>> open("x.hpp").read()[26:32-len(spelling)].strip()
'Test'
```
Diagnostic would also need to be manually removed, but this could work without tmp file.